### PR TITLE
Helm test

### PR DESCRIPTION
**Pull request by denisok**
https://jira.percona.com/browse/PMM-9985

This workflow would be triggered from Jenkins and would run Helm or other test in a future

Sequence of landing:

1. https://github.com/Percona-Lab/pmm-submodules/pull/2583
2. https://github.com/percona/pmm-qa/pull/488
3. https://github.com/Percona-Lab/jenkins-pipelines/pull/1601

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2583-dd6c320
client docker - perconalab/pmm-client-fb:PR-2583-dd6c320
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2583-dd6c320.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2583-dd6c320&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2583-dd6c320.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/10370/

**Comment by puneet0191** [comment]
@denisok I will change submodules default branch to the one you sent on Jenkins, to see how trigger works, this is very cool stuff, thank you

**Comment by denisok** [comment]
@puneet0191 I think it is easier to land this and then debug locally with curl and test PRs. Anyway it will not run by default, ether manual trigger or API call.
I am not sure myself how it would report statuses and where :) 

If it works as expected we also could add jobs to report statuses for RC build - filtering by branch.

File: .github/workflows/jenkins-dispatch.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        with:
+          node-version: '14'
+      - run: npm install -g bats
+
+  minikube_start:
+    needs: setup
+    runs-on: ubuntu-latest
+    steps:
+      - run: |
+          ./pmm-qa/pmm-tests/install_k8s_tools.sh --minikube
```
Comment (puneet0191) [inline .github/workflows/jenkins-dispatch.yml]: where do you exactly checkout pmm-qa?

Comment (denisok) [reply .github/workflows/jenkins-dispatch.yml]: submodules: 'recursive' but you are right, changing path to correct one

**Review by puneet0191 — APPROVED** [review]

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2583-60908fa
client docker - perconalab/pmm-client-fb:PR-2583-60908fa
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2583-60908fa.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2583-60908fa&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2583-60908fa.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/10404/

**Pull request merged**